virttest/storage.py

```diff
@@ -557,18 +613,66 @@ def __init__(self, params, root_dir, tag):
                                                               root_dir)
             self.snapshot_format = ss_params.get("image_format")
 
-        self.image_access = retrieve_access_info(self.tag, self.params)
+        self.image_access = ImageAccessInfo.access_info_define_by_params(
+                                                   self.tag, self.params)
+
+    def need_access_info(self, image=None):
```

Comment:
1. should be a private method
2. as it is requesting the auth information, suggest the name to be `_need_auth_info`
3. should be moved into `qemu_storage`

Reply:
done